Replicator/Replicator.cc

```diff
@@ -851,7 +849,8 @@ namespace litecore { namespace repl {
                     SubReplicator& sub = _subRepls[i];
                     Dict dict = checkpointArray[i].asDict();
                     if (!dict) {
-                        auto error = C4Error::printf(LiteCoreDomain, kC4ErrorNotFound,
+                        // Make it fatal error, UnexpectedError.
+                        auto error = C4Error::printf(LiteCoreDomain, kC4ErrorUnexpectedError,
```

Comment:
Basically the collection is not found on the remote side. I think NotFound is suitable unless NotFound is not a permanent error?

Reply:
This is not the reason I changed the error code. It is because I have to stop the replicator and NotFound won't stop the  replicator.

Reply:
Do you mean the replicator keeps retrying?

Reply:
yes